samples/notifications.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic operations on files with
+ * the Google Cloud Storage API.
+ *
+ * For more information, see the README.md under /storage and the documentation
+ * at https://cloud.google.com/storage/docs.
+ */
+
+'use strict';
+
+function createNotification(bucketName, topic) {
```

Comment:
Samples LGTM, but we'll want tests for them. They're pretty simple tests to write, they just exercise the CLI. See https://github.com/googleapis/nodejs-storage/blob/master/samples/system-test/buckets.test.js for an example. If the tests require that a topic be created and then removed at the end of the test, then `samples/package.json` will need to depend on `@google-cloud/pubsub`.